### PR TITLE
HW2_Ihor_Holubtsov

### PR DESCRIPTION
**Pull request by Swipp1233**
HW2 + refactor

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
 
 module.exports = defineConfig({
+
   e2e: {
+    chromeWebSecurity: false,
+    baseUrl: 'http://localhost:3000',
+    watchForFileChanges: false,
+    numTestsKeptInMemory: 50,
+    specPattern: 'cypress/e2e/*.{js,jsx,ts,tsx}',
+    retries: {
```
Comment (ievgendco) [inline cypress.config.js:11]: In this case you could use retries: 2

File: cypress/e2e/Sign_Up_page_checks.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import {sign_in_page} from "../selectors/sign_in_page_selectors";
+import {sign_up_page} from "../selectors/sign_up_page_selectors";
+
+
+describe('cypress_loginPage_cases', () => {
+  before(() => {
+    cy.visit('/')
+    cy.viewport(1920, 1080)
```
Comment (ievgendco) [inline cypress/e2e/Sign_Up_page_checks.js:8]: It should be moved to the config file

File: cypress/e2e/Welcome_checks.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  it('check "Remember me" checkbox', () => {
+    cy.get(sign_in_page.remember_me_checkbox).click().should('be.checked').click().should('be.not.checked')
+  })
+
+  it('should show disabled by default sign in btn', () => {
+    cy.get(sign_in_page.login_field).should('have.value', '')
+    cy.get(sign_in_page.sign_in_button).should('be.disabled')
+  })
+
+  it('should have \'Don\'t have an account? Sign Up\' clickable link under \'Sign in\' btn', () => {
```
Comment (ievgendco) [inline cypress/e2e/Welcome_checks.js]: You can use double quotes, in this case, there is no need to use the escape character

File: cypress/e2e/Welcome_checks.js [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    cy.go('back')
+  })
+})
+
+  //SIGNUP, LOGIN, ONBOARDING AND LOGOUT FLOWS
+describe('Checking signup, login, onboarding and logout flows', () => {
+
+  before(() => {
+    cy.visit('/')
+    cy.viewport(1920, 1080)
```
Comment (ievgendco) [inline cypress/e2e/Welcome_checks.js:68]: It should be moved to the config file

File: cypress.config.js (hunk 1 above)
Comment (Swipp1233) [reply cypress.config.js:11]: +

File: cypress/e2e/Sign_Up_page_checks.js (hunk 2 above)
Comment (Swipp1233) [reply cypress/e2e/Sign_Up_page_checks.js:8]: fixed

File: cypress.config.js (hunk 1 above)
Comment (Swipp1233) [reply cypress.config.js:11]: +

Comment (Swipp1233) [reply cypress.config.js:11]: +

File: cypress/e2e/Sign_Up_page_checks.js (hunk 2 above)
Comment (Swipp1233) [reply cypress/e2e/Sign_Up_page_checks.js:8]: fixed

File: cypress/e2e/Welcome_checks.js (hunk 4 above)
Comment (Swipp1233) [reply cypress/e2e/Welcome_checks.js:68]: fixed

File: cypress/e2e/Welcome_checks.js (hunk 3 above)
Comment (Swipp1233) [reply cypress/e2e/Welcome_checks.js]: fixed

**Review by anatoly-devico — APPROVED** [review]
Approved

**Pull request merged**